src/cache/surface_albedo.jl

```diff
@@ -0,0 +1,195 @@
+# we are ignoring the volume reflectance for now
+abstract type SurfaceAlbedoModel end
+
+"""
+    Base.@kwdef struct ConstantAlbedo{FT} <: SurfaceAlbedoModel
+
+A constant surface albedo model. The default value is 0.38. It is used purely for idealized experiments.
+"""
+Base.@kwdef struct ConstantAlbedo{FT} <: SurfaceAlbedoModel
+    α::FT = 0.38
```

Comment:
Following the current default.

Reply:
Is the default referred to in the paper? If not, can we please include the source? (Looking over Jin et. al now, will review the docs as I do this)

Reply:
Ahh, it's not following the paper. It's following the original ClimaAtmos default value for the contant albedo. Maybe it was 0.38 because it is approximately the average of the geometric and bond albedos of Earth? Tbh I would expect the bond albedo, but since the constant albedo is only used for idealized simulations, maybe it doesn't matter too much either way. I kept the same value as on main, so as to confirm that the regression tests are still passing.

Reply:
@szy21 , do you know why we chose 0.38?

Reply:
It's from [this](https://journals.ametsoc.org/view/journals/clim/21/15/2007jcli2065.1.xml) paper, to achieve a relatively reasonable global mean energy balance. We should make it a parameter in the future, but this doesn't matter much.

Reply:
Makes sense, thanks!